docs/content/latest/benchmark/tpcc-ysql/10000-wh/tpcc-ysql.md

```diff
@@ -58,16 +55,16 @@ Once the loading is done we need to enable the foreign keys that were disabled t
   </tbody>
 </table>
 
-### TPC-C Execute Phase
+## TPC-C Execute Phase
 
-Before beginning the execution we will have to move all the tablet leaders out of the node containing the master leader. This can be achieved by:
+Before starting the execution, you have to move all the tablet leaders out of the node containing the master leader by running the following command:
 
 ```sh
 ./yb-admin --master_addresses <master-ip1>:7100,<master-ip2>:7100,<master-ip3>:7100 change_leader_blacklist ADD <master-leader-ip>
 ```
 
-Also make sure that the ips used in the execution phase doesn't include the master-leader-ip.
-You can then run the workload against the database as follows from each of the client as follows:
+Make sure that the IPS used in the execution phase does not include the `master-leader-ip`.
+You can then run the workload against the database from each of the client:
 
```

Comment:
each of the clients:

Reply:
Oops, I wanted to change it to "each client".